optuna/samplers/_cmaes.py

```diff
@@ -218,6 +221,18 @@ def objective(trial):
                 versions without prior notice. See
                 https://github.com/optuna/optuna/releases/tag/v3.1.0.
 
+        lr_adapt:
+            If this is :obj:`True`, CMA-ES with learning rate adaptation is used.
+            This algorithm focuses on working well on multimodal and/or noisy problems
+            with default settings.
+            Currently, this option cannot be used with ``use_separable_cma=True`` nor
```

Comment:
I would say here we should use "or" since the sentence starts with "cannot be used ...". Or  we can say "Currently, this option can be used neither with ... nor ...".

```suggestion
            Currently, this option cannot be used with ``use_separable_cma=True`` or
```